lib/internal/test_runner/coverage.js

```diff
@@ -138,9 +139,9 @@ class TestCoverage {
     return lines;
   }
 
-  summary() {
+  summary(excludeFiles) {
```

Comment:
I'm not a big fan of passing `excludeFiles` around all over the place like this. It's also confusing because we have `excludeGlobs` already, but this seems to be handled totally differently.

Reply:
Me neither :-/ I will rethink this.

Reply:
IMO it should be in `this.options`, originally set from when the files are detected. That will remove all of this extra logic